File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
       for(int i = 0; i < nums.size();i++)
       {
            int curr = nums[i];
            for(int j = i + 1; j < nums.size();j++)
            {
                if(curr == nums[j])   
                {
                    return true;
                }

            }
       }
       return false;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        output = {}

        for index, value in enumerate(nums):
            difference = target - value
            if difference in output:
                return [output[difference], index] 
            output[value] = index
            
        
       
            